wondrous-app/components/Onboarding/utils.tsx

```diff
@@ -1,10 +1,22 @@
-export const handleUserOnboardingRedirect = (user, router) => {
-  const { collabInvite } = router.query;
+export const handleUserOnboardingRedirect = (user, router, inviteToken = null, inviteType = null) => {
+  const { collabInvite, token, type } = router.query;
 
-  const defaultRoute = collabInvite ? `/invite/collab/${collabInvite}` : '/mission-control';
+  const discordOrRouterToken = inviteToken || token;
+
+  const discordOrRouterType = inviteType || type;
+
+  let defaultRoute = collabInvite ? `/invite/collab/${collabInvite}` : '/mission-control';
+
+  if (discordOrRouterToken) {
+    defaultRoute = `/invite/${discordOrRouterToken}`;
+  }
+
+  if (discordOrRouterType === 'pod') {
+    defaultRoute = `/invite/pod/${discordOrRouterToken}`;
+  }
 
   const collabInviteQuery = collabInvite ? `?collabInvite=${collabInvite}` : '';
-  if (user?.signupCompleted) {
+  if (user?.signupCompleted || inviteToken) {
```

Comment:
So this redirects them back to the pod invite page?

Reply:
yeah basically, to avoid managing redirects in multiple places

Here's a better loom

https://www.loom.com/share/c8c978b528c74c53bce6c1eef05567db

Reply:
Nice think this works still